content/guides/virtualizing/vmware-workstation.md

```diff
@@ -1,131 +1,111 @@
 +++
 type = "article"
-title = "Virtualizing Haiku in VMware-Workstation "
+title = "Virtualizing Haiku in VMware Workstation"
 date = "2016-12-22T18:11:53.000Z"
 tags = []
 +++
 
-The required ISO file can be found at on the "Get Haiku" page of this website which is located [here](https://www.haiku-os.org/get-haiku). VMware Player is available for free on their website which can be found [here](https://www.vmware.com/products/player/). There is also VMware Workstation, which is similar but has more features and is commercial. This means you have to pay for this version, but it does offer a 30 days trial. However you could just stick to the free one _(VMware Player)_ as you most likely will not need the extra features. You can both read about the commercial version's extra features and download it from its website [here](https://www.vmware.com/products/workstation/overview.html)
+Virtual instances of operating systems are perfect for all kinds of testing purposes that need to be done in a safe and isolated environment. Installing Haiku in a virtual machine is a solution for people who do not want to install it on their physical computers, but wish to become familiar with it.
 
-##### Go to section:
+This How-To guide will describe the process of running Haiku on virtual machine (VM) using VMWare Workstation 14 and an Anyboot image file.
 
-*   [Running Haiku from a VM image.](#part_vmimage)
-*   [Installing and running Haiku from an ISO image.](#part_iso)
-*   [Additional Steps.](#part_additional)
-*   [Troubleshooting.](#part_trouble)
+The Anyboot image file can be obtained [here](http://localhost:8080/get-haiku). VMWare Player is available from VMWare for free [on their website](https://www.vmware.com/products/player/). There is also VMware Workstation, which is similar but has more features and is commercial. This means you have to pay for this version, but it does offer a 30 days trial. However you could just stick to the free one _(VMware Player)_ as you most likely will not need the extra features. You can both read about the commercial version's extra features and download it from its website [here](https://www.vmware.com/products/workstation/overview.html)
```

Comment:
http://localhost:8080/get-haiku   :)

Reply:
Ah, brain burp. Fixed.

Reply:
Doesn't seem to be fixed...?

Reply:
Apparently I should not edit while on the way to school. Should be actually fixed now.